ansible/asqatasun/templates/xvfb.j2

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
If I see correctly, this `xvfb` is just identical to the original one. Do you think we could make a symlink instead of duplicating the file ? This would greatly improve maintainability. (I know Docker absolutely wants a regular file in its directory, I don't know if Ansible is more tolerant).